monix-execution/shared/src/main/scala/monix/execution/misc/Local.scala

```diff
@@ -210,24 +261,29 @@ final class Local[A](default: () => A) {
   /** Execute a block with a specific local value, restoring the
     * current state upon completion.
     */
+  @deprecated("Binds on Local prevent propagation of writes to other locals. Use Local.isolate directly", "3.0.0-RC3")
```

Comment:
I think there is also `bind` in `Local` companion object, should we deprecate/remove it too?

Reply:
Binding on whole context is fine. The problem is that binding one variable implicitly binds whole context.